kuma/static/js/wiki-helpful-survey.js

```diff
@@ -0,0 +1,37 @@
+(function($, mdn) {
+    'use strict';
+
+    var $survey;
+    var $votes;
+
+    // survey won't do much good if we can't store the data
+    // is this the best way to check for the existence of GA?
+    if (window.ga) {
+        $survey = $('#helpful-survey');
+        $votes = $survey.find('.helpful-survey-vote');
+
+        $survey.removeClass('hidden');
+
+        $votes.on('click', function() {
+            // hey, so which one was clicked?
+            var isHelpful = $(this).hasClass('helpful-survey-yes');
+
+            // store the vote in ye olde google
+            /* am i doing it right?
+            mdn.analytics.trackEvent({
```

Comment:
Correct technique. Let's call the category `helpful2`. Action is good. Please make the label `Yes` or `No`, that seems to be how we record binary data in MDN's GA account.